### DISCUSSION TITLE
StatusBarBehavior StatusBarColor DynamicResource binding not supported?

### DISCUSSION THREAD

**Discussion by Cybrosys** [CommunityToolkit/Maui#1054] [Q&A]
Hello,

Could someone explain why this does not work?
```
<ContentPage.Behaviors>
    <toolkit:StatusBarBehavior StatusBarColor="{DynamicResource Tertiary}" />
 </ContentPage.Behaviors>
```

I am explicitly after the DynamicResource feature to support theming, binding it to a StaticResource works. Is using a DynamicResource binding not supported on things put inside .Behaviors or what am I missing?

**Comment by TheCodeTraveler** [comment]
Could you share a [reproduction sample](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#bug-reproduction) demonstrating this?

**Reply by Cybrosys** [reply]
Of course, here:
https://github.com/Cybrosys/Sandbox/tree/master/MauiAppStatusBarColorDynamicResource

I'm testing on Android.

**Reply by TheCodeTraveler** [reply]
Thanks! I'm able to reproduce it on iOS.

Could you open a [Bug Report](https://github.com/CommunityToolkit/Maui/issues/new?assignees=&labels=bug%2Cunverified&template=bug_report_template.yml&title=%5BBUG%5D+)?

**Reply by Cybrosys** [reply]
Will do, I'll post a link here to it once I'm done.

**Reply by Cybrosys** [reply]
https://github.com/CommunityToolkit/Maui/issues/1056

**Comment by CliffAgius** [comment]
Closed and answered
